client/package.json

```diff
@@ -107,6 +107,7 @@
     "json-loader": "^0.5.4",
     "karma": "^1.2.0",
     "karma-chrome-launcher": "^2.0.0 ",
+    "karma-firefox-launcher": "^1.0.0",
```

Comment:
You need Firefox in Travis for the e2e test, since the installed Chrome version in Travis is too old for the e2e tests. Therefore, Firefox is used for the e2e tests. 
Chromium is used for the Jasmine tests, since one test could not be executed with Firefox. No idea why.